python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```
@@ -1425,6 +1429,268 @@ keys:
+                tx multicast: Allow multicast packets only to go out of the source interface.
+              valid_values:
+                - tx
+                - tx multicast
+            access_vlan:
+              type: int
+              convert_types:
+                - "str"
+              min: 1
+              max: 4094
+              description: |-
+                Set VLAN when interface is in access mode.
+                Warning: This should not be combined with `ethernet_interfaces[].mode = access/dot1q-tunnel`.
+            phone:
+              type: dict
+              keys:
+                vlan:
+                  type: int
+                  min: 1
+                  max: 4094
+                  convert_types:
+                    - str
+                  description: |-
+                    Warning: This should not be combined with `ethernet_interfaces[].phone.vlan`.
+                trunk:
+                  type: str
+                  description: |-
+                    Warning: This should not be combined with `ethernet_interfaces[].phone.trunk`
+                  valid_values: ["tagged", "tagged phone", "untagged", "untagged phone"]
+            mode:
+              type: str
+              valid_values: ["access", "dot1q-tunnel", "trunk", "trunk phone"]
+              description: |-
+                Warning: This should not be combined with `ethernet_interfaces[].mode`
+            pvlan_mapping:
+              type: str
+              convert_types:
+                - int
+              description: |-
+                Secondary VLAN IDs of the private VLAN mapping.
+                Warning: This should not be combined with `ethernet_interfaces[].pvlan_mapping`.
+            vlan_translations:
+              description: |-
+                Warning: This should not be combined with `ethernet_interfaces[].vlan_translations`.
+              type: dict
+              keys:
+                direction_in:
+                  type: list
+                  description: Map ingress packets only.
+                  items:
+                    type: dict
+                    keys:
+                      required:
+                        type: bool
+                        description: Drop the packets that do not match any VLAN mapping.
+                      from:
+                        type: int
+                        convert_types:
+                          - str
+                        min: 1
+                        max: 4094
+                        description: VLAN ID or range of VLAN IDs to map from.
+                      to:
+                        type: int
+                        convert_types:
+                          - str
+                        min: 1
+                        max: 4094
+                        description: VLAN ID to map to.
+                      dot1q_tunnel:
+                        type: bool
+                      inner_from:
```

```suggestion
                      inner_vlan:
```
This key name looks better.

---

but we have `inner_to` also

---

it is for direction_out.

---

changed it to `inner_vlan_from` and `inner_vlan_to`